packages/edit-post/src/editor.js

```diff
@@ -92,7 +92,10 @@ function Editor( {
 				hiddenBlockTypes: getHiddenBlockTypes(),
 				blockTypes: getBlockTypes(),
 				template:
-					supportsTemplateMode && isViewable && canEditTemplate
+					supportsTemplateMode &&
+					isViewable &&
+					canEditTemplate &&
+					currentPost.postType !== 'wp_template'
```

Comment:
That's weird, I'd have expected isViewable to be false for templates.

Reply:
It does seem to be `false`. What is `isViewable`? 😄

Reply:
I think isViewable means a post type is viewable in the frontend. It's the closest thing to "postType has template" that I can think of. So maybe this extra check is useless.